egs/aishell/v1/local/produce_trials.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Bengu Wu
+# Apache 2.0.
```

Comment:
Since this script defines the evaluation you're going to perform, it would be great to get some comment here, describing how the trials are generated.